### DISCUSSION TITLE
放一个停云（米游社看到的非常可爱∗︎˚(* ˃̤൬˂̤ *)˚∗︎）

### DISCUSSION THREAD

**Discussion by qwertyuiopa1231111** [share121/inter-knot#428] [其他]
![1723233953982657be9e93aa347125eaf1c3389886a7731014a09850f72f40684c7834f18ff49 0](https://github.com/user-attachments/assets/488be375-06cd-4032-bf3b-746ca2a11bea)

**Comment by SaKura-071721** [comment]
想摸

**Comment by opqphp** [comment]
求画师<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Comment by SKWTEA** [comment]
好可爱啊👍

**Comment by 15666851105** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/d41b52daa44ef813710cf4b482a256b2.png" alt="三月七_点赞" width="75" >

**Comment by wzyaeu** [comment]
卡哇伊！

**Comment by BiancaAtagina** [comment]
好看好看

**Comment by Ugly745** [comment]
可爱捏

**Reply by opqphp** [reply]
？！！还有人在
